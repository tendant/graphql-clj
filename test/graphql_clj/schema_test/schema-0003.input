type Person {
name: String
age: Int
}

type Photo {
height: Int
width: Int
}
