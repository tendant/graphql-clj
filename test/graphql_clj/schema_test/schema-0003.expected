{:objects
 {:Person {:fields {:name {:type String}, :age {:type Int}}},
  :Photo {:fields {:height {:type Int}, :width {:type Int}}}}}
